DependencyInjection/HWIOAuthExtension.php

```diff
@@ -108,17 +108,27 @@ public function createResourceOwnerService(ContainerBuilder $container, $name, a
         if (isset($options['service'])) {
             // set the appropriate name for aliased services, compiler pass depends on it
             $container->setAlias('hwi_oauth.resource_owner.'.$name, $options['service']);
-        } else {
-            $type = $options['type'];
-            unset($options['type']);
 
+            return;
+        }
+
+        $type = $options['type'];
+        unset($options['type']);
+
+        // handle external resource owners with given class
+        if (isset($options['class'])) {
```

Comment:
How to do so? Trying to instantiate it with fake data to see if implements `ResourceOwnerInterface`? Isn't it overkill?